src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -554,6 +574,12 @@ ErrorIfSubqueryNotSupported(Query *subqueryTree)
 		errorDetail = "Subqueries with offset are not supported yet";
 	}
 
+	if (QueryReferencesView(subqueryTree))
+	{
+		preconditionsSatisfied = false;
```

Comment:
Aren't there classes of this that we actually would support? I'm not saying we have to, but I think if so this should carry a comment saying that we just don't bother to figure out which are supported.